.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish new version
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    name: Publish
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          persist-credentials: false
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+      - name: Semantic Release
+        uses: BrightspaceUI/actions/semantic-release@master
+        with:
+          GITHUB_TOKEN: ${{ secrets.D2L_GITHUB_TOKEN }}
```

Comment:
Double-check: plan is to keep this rotated once that infrastructure is ready?

Reply:
Correct.

Reply:
I believe so. I had @dlockhart add the secret, and I assume there's some infrastructure in the works to rotate it in the works